packages/rule-typescript-config/docs/target.md

```diff
@@ -0,0 +1,113 @@
+# `target`
+
+`typescript-config/target` takes into account your `sonarwhal`'s `browserslist`
+configuration and warns you if your `target` (`es3`, `es2015`, etc.) is
+appropriate.
+
+### Why is this important?
+
+Not having the right `ES` target can increase the size of your bundle. At the same
+time, having a `ES` target that not all the browsers you care support can create
+issues with your users.
+
+### What does the rule check?
+
+This rule checks the `compilerOptions` property `target` is the appropriate based
+on your `browserslist` configuration.
+
+The matrix is as follows:
+
+| Target | Chrome | Edge | Firefox | IE | Safari |
+| ------ | ------ | ---- | ------- | -- | ------ |
+|  ES3   |   ✔    |  ✔  |    ✔    | ✔ |    ✔   |
+|  ES5   |   57   |  12  |   45    | 9 |    8    |
```

Comment:
I'm sure the values for chrome are not the right ones. Need to recheck the kangax tables and update this accordingly.